br/pkg/lightning/importer/table_import.go

```diff
@@ -744,18 +744,18 @@ ChunkLoop:
 	// Report some statistics into the log for debugging.
 	totalKVSize := uint64(0)
 	totalSQLSize := int64(0)
-	typ := "bytes"
+	logKeyName := "bytes"
 	for _, chunk := range cp.Chunks {
 		totalKVSize += chunk.Checksum.SumSize()
 		totalSQLSize += chunk.UnfinishedSize()
 		if chunk.FileMeta.Type == mydump.SourceTypeParquet {
-			typ = "rows"
+			logKeyName = "rows"
 		}
 	}
 
 	err = chunkErr.Get()
 	logTask.End(zap.ErrorLevel, err,
-		zap.Int64(fmt.Sprintf("read(%s)", typ), totalSQLSize),
+		zap.Int64(fmt.Sprintf("read(%s)", logKeyName), totalSQLSize),
```

Comment:
we can include the read in the `logKeyName` so no need `sprintf`, but it's ok now.